portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACApi.macro

```diff
@@ -32,24 +32,36 @@ definition {
 			expected = ${expected});
 	}
 
-	macro createNewExportSchedule {
+	macro createDataReport {
 		var analyticsCloudURL = PropsUtil.get("analytics.cloud.url");
 
-		var curl = '''
-			${analyticsCloudURL}/api/reports/export/${type}\?fromDate=${startDate}T${startTime}Z\&toDate=${endDate}T${endTime}Z
-				-H "Authorization: Bearer ${token}"
-		''';
+		if (isSet(type)) {
+			var curl = '''
+				${analyticsCloudURL}/api/reports/${type} -H "Authorization: Bearer ${token}"
+			''';
+		}
+		else if (isSet(filter)) {
+			var curl = '''
+				${analyticsCloudURL}/api/reports/${type}\?${filter}=${filterValue} -H "Authorization: Bearer ${token}"
+			''';
```

Comment:
Hey @ViniLopes20, thanks for working on this! I have a question:
I know that we are not obligated to provide a type for the curl, but can I use this curl **only providing the filter parameter**?

Reply:
Hey @manuferreira, when you don't use the `type` in curl you can only get a single response via API which is the list of endpoints available to be used (the list displays the endpoint for each `type` and the `filters` available for each one), so there's no way you can use the `filter` in curl if you don't define the `type` you're going to use because that would only return the list of endpoints

Reply:
Thanks for the clarification!